podcasts/mit-sloan-sports-analytics-conference.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 rss_url: https://feed.podbean.com/ssac/feed.xml
-topics: Physical Education and Recreation
+topics: Physical Education and Recreation, Business
```

Comment:
Actually, there's a bug (that Anastasia is working on as we speak) where you can't have whitespace between list items. It will parse it as " Business", which will be different than "Business"

Reply:
... actually... this could be a good test case for this PR: https://github.com/mitodl/open-discussions/pull/3000.

Reply:
That PR has merged, so let's try this.